.github/workflows/buildAndTest.yml

```diff
@@ -118,8 +118,8 @@ jobs:
 
   # Build the LLVM submodule then cache it. Do not rebuild if hit in the
   # cache.
-  build-llvm:
-    name: Build LLVM
+  build-llvm-clang:
```

Comment:
IIRC you can avoid the yaml code duplication with a matrix (?) build.

Reply:
Thanks!